extras/bindings/c/src/SolverInterfaceC.cpp

```diff
@@ -166,6 +166,14 @@ int precicec_getMeshID(const char *meshName)
   return impl->getMeshID(stringMeshName);
 }
 
+void precicec_getMeshIDs(int *ids)
```

Comment:
Sadly not safe.

Reply:
But it is consistent with the rest of our C bindings. Don't we have the same problem there?

If we want to keep this method, let's keep it consistent to the rest. Otherwise, let's rewrite the C bindings to be safer.